### DISCUSSION TITLE
image_thumb field not working

### DISCUSSION THREAD

**Discussion by tprus-luc** [CollectionBuilder/collectionbuilder.github.io#139] [Q&A]
I noticed a few minutes ago that CB is using the original items as thumbnails. Even though I have ImageMagick installed on my dev machine, I didn't take any chances. I quickly created batch resized copies of every image to a new "objects/thumbnails" directory. I then created a new `image_thumb` column at the end of my CSV file. In the "image_thumb" column I copied over the file names and prepended my relative thumbnails path to the file names, so it reads something like "/objects/thumbnails/file001.jpg" for a filename of "file001.jpg".

However, no matter long I wait or how many times I restart the Jekyll server, the Browse and Timeline pages continue to use the original items as thumbnails rather than the thumbnails.

Also, perhaps related, my one video item also refuses to load its thumbnail, which is also specified as "/objects/thumbnails/my_video.jpg" in the image_thumb column. CB simply uses the default video icon for a filename of "myvideo.mp4". This acceptable, but not ideal.

**Comment by evanwill** [comment]
Are you using collectionbuilder-gh instead of collectionbuilder-csv template?
Could you provide a link to your project?

**Reply by tprus-luc** [reply]
I am using collectionbuilder-gh, I wasn't aware of the difference until I was already deep into my project. For privacy reasons, I'm developing my project on my own Windows PC.
